1209/FFFF/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: Smart House
+owner: LifeToDevice

```

Comment:
This needs to match your folder name in org/ so either To needs to become 2 or vice versa.